README.md

```diff
@@ -11,36 +106,100 @@ is supported where the Avro schemas are obtained from a configured
 
 An arbitrary Zed script can be applied to the Zed records in either direction.
 
-Zinger formats records received from Kafka using the Zed envelope
+The Zed pool used by `zinger` must have its pool key set to `kafka.offset` in
+descending order.  `zinger` will detect and report an error if syncing
+is attempted using a pool without this configuration.
+
+Each Kafka topic must have a single partition as the system relies upon
+the offset to indicate the FIFO order of all records.
+
+### Sync From
+
+`zinger sync from` formats records received from Kafka using the Zed envelope
 ```
 {
-        kafka: {topic:string,offset:int64,partition:int32},
-        record: {...}
+  kafka: {topic:string,partition:int64,offset:int64,input_offset:int64},
+  key: {...}
+  value: {...}
 }
 ```
-where `...` indicates the Zed record that results from the configured Zed script
-applied to the decoded Avro record.
-If there is no such script, the record is simply the verbatim result obtained
-from decoding Avro into Zed.
+where the `key` and `value` fields represent the key/value data pulled from
```

Comment:
```suggestion
where the `key` and `value` fields represent the key and value data pulled from
```

Reply:
I like key/vaule because it emphasizes the Kafka data model... changing to "key/value data pair"